base/util.go

```diff
@@ -658,7 +658,7 @@ func GetGoCBBucketFromBaseBucket(baseBucket Bucket) (bucket CouchbaseBucketGoCB,
 	case *CouchbaseBucketGoCB:
 		return *baseBucket, nil
 	default:
-		return CouchbaseBucketGoCB{}, fmt.Errorf("baseBucket %v was not a CouchbaseBucketGoCB.  Was type: %T", baseBucket, baseBucket)
+		return CouchbaseBucketGoCB{}, RedactErrorf("baseBucket %v was not a CouchbaseBucketGoCB.  Was type: %T", MD(baseBucket), MD(baseBucket))
```

Comment:
We don't want to convert the 2nd `baseBucket` into Metadata as we're capturing the type (`%T) here.